opendebates/views.py

```diff
@@ -239,7 +239,12 @@ def vote(request, id):
     )
     if created:
         # update the DB with the real tally
-        Submission.objects.filter(id=id).update(votes=F('votes')+1)
+        Submission.objects.filter(id=id).update(
+            votes=F('votes')+1,
+            local_votes=F('local_votes')+(
+                1 if voter.state and voter.state == get_local_votes_state()
+                else 0)
+        )
```

Comment:
Yeah, I'm okay with that -- I can't think of any reason that setting would change during a voting period (except if the admins just forget to configure it correctly in the first place) and in the worst case I guess we could just do an ad hoc data migration to recalculate the tallies.